processors/code/build.go

```diff
@@ -125,6 +128,17 @@ func gatherRequirements(envModel *models.Env, containerID string) error {
 	return nil
 }
 
+// populate the build triggers so we can know next time if a change has happened
+func populateBuildTriggers(envModel *models.Env) {
+	if envModel.BuildTriggers == nil {
+		envModel.BuildTriggers = map[string]string{}
+	}
+	box := boxfile.New([]byte(envModel.UserBoxfile))
+	for _, trigger := range box.Node("run.config").StringSliceValue("build_triggers") {
+		envModel.BuildTriggers[trigger] = util.FileMD5(trigger)
```

Comment:
this should probably try to find the file from the absolute path prefixing the app dir, in the event that nanobox was run in a subfolder of this project.